File: sdkwork-backend-sdk-csharp/Models/PlusGameTournamentBatchStatusPlayerForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class PlusGameTournamentBatchStatusPlayerForm
    {
        public int? UserId { get; set; }
        public string? Status { get; set; }
        public int? RankNo { get; set; }
        public double? ScoreValue { get; set; }
        public int? TotalMatchCount { get; set; }
        public int? WinCount { get; set; }
        public int? LoseCount { get; set; }
    }
}

File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusGameTournamentBatchStatusPlayerForm.kt
package com.sdkwork.backend

data class PlusGameTournamentBatchStatusPlayerForm(
    val userId: Int? = null,
    val status: String? = null,
    val rankNo: Int? = null,
    val scoreValue: Double? = null,
    val totalMatchCount: Int? = null,
    val winCount: Int? = null,
    val loseCount: Int? = null
)
